docs/nightly/en/user-guide/concepts/data-model.md

```diff
@@ -95,4 +95,4 @@ GreptimeDB is designed on top of Table for the following reasons:
   The multi-value model is used to model data sources, where a metric can have multiple values represented by fields.
   The advantage of the multi-value model is that it can write or read multiple values to the database at once, reducing transfer traffic and simplifying queries. In contrast, the single-value model requires splitting the data into multiple records. Read the [blog](https://greptime.com/blogs/2024-05-09-prometheus) for more detailed benefits of multi-value mode.
 
-GreptimeDB uses SQL to manage table schema. Please refer to [table management](../table-management.md) for more information. However, our definition of schema is not mandatory and leans towards a **schemaless** approach, similar to MongoDB. For more details, see [Automatic Schema Generation](../write-data/overview.md#automatic-schema-generation).
+GreptimeDB uses SQL to manage table schema. Please refer to [table management](../table-management.md) for more information. However, our definition of schema is not mandatory and leans towards a **schemaless** approach, similar to MongoDB. For more details, see [Automatic Schema Generation](../ingest-data/overview.md#automatic-schema-generation).
```

Comment:
Always use the `/user-guide/table-management.md` and `/user-guide/ingest-data/overview...` is a good practice. The relative path is confused sometimes.

Reply:
This PR will not check all path writing methods, which will be modified with future PRs.